packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -541,11 +545,23 @@ class ConstrainedBox extends OneChildRenderObjectWidget {
   }
 }
 
+/// Sizes itself to a fraction of the total available space.
+///
+/// See [RenderFractionallySizedBox] for details.
 class FractionallySizedBox extends OneChildRenderObjectWidget {
   FractionallySizedBox({ Key key, this.width, this.height, Widget child })
     : super(key: key, child: child);
 
+  /// If non-null, the factor of the incomming width to use.
+  ///
+  /// If non-null, the child is given a tight width constraint that is the max
+  /// incomming width constraint multipled by this factor.
```

Comment:
maybe you should just do a grep for "incomming"... I'm not going to point out the rest. :-)